docs/content/latest/architecture/transactions/read-committed.md

```diff
@@ -0,0 +1,785 @@
+---
+title: READ COMMITTED isolation level
+headerTitle: READ COMMITTED isolation level
+linkTitle: READ COMMITTED isolation level
+description: Details about READ COMMITTED isolation level
+aliases:
+  - /architecture/transactions/read-committed/
+menu:
+  latest:
+    identifier: architecture-read-committed
+    parent: architecture-acid-transactions
```

Comment:
put this page under Transaction isolation levels page rather than beside it.

Reply:
done

Reply:
What I mean is can you make isolation-levels the parent